Apps/WebClient/src/NewClientApp/src/views/RegistrationView.vue

```diff
@@ -0,0 +1,388 @@
+                    id="emailCheckbox"
+                    v-model="isEmailChecked"
+                    data-testid="emailCheckbox"
+                    @change="onEmailOptout($event)"
+                    label="Email Notifications"
+                ></v-checkbox>
+                <em class="text-body-2">
+                    Receive application and health record updates
+                </em>
+                <v-text-field
+                    id="emailInput"
+                    v-model="v$.email.$model"
+                    :disabled="!isEmailChecked"
+                    data-testid="emailInput"
+                    placeholder="Your email address"
+                    type="email"
+                    :error-messages="emailErrorMessages"
+                />
+                <v-text-field
+                    id="emailConfirmationInput"
+                    v-model="v$.emailConfirmation.$model"
+                    :disabled="!isEmailChecked"
+                    data-testid="emailConfirmationInput"
+                    placeholder="Confirm your email address"
+                    type="email"
+                    :error-messages="emailConfirmationErrorMessages"
+                />
+            </div>
+            <v-divider thickness="3" class="mt-6"></v-divider>
+            <!-- SMS section -->
+            <div>
+                <v-checkbox
+                    id="smsCheckbox"
+                    v-model="isSMSNumberChecked"
+                    @change="onSMSOptout($event)"
+                    label="Text Notifications"
+                ></v-checkbox>
+                <em class="text-body-2">
+                    Receive health record updates only
+                </em>
+                <v-text-field
+                    id="smsNumberInput"
+                    v-model="v$.smsNumber.$model"
+                    v-maska:[smsMaskaOptions]
+                    :disabled="!isSMSNumberChecked"
+                    data-testid="smsNumberInput"
+                    placeholder="Your phone number"
+                    type="tel"
+                    :error-messages="smsNumberErrorMessages"
+                />
+            </div>
+            <div
+                v-if="!isEmailChecked && !isSMSNumberChecked"
+                class="font-weight-bold text-primary"
+            >
+                <v-icon
+                    aria-hidden="true"
+                    class="mr-2"
+                    icon="fas fa-exclamation-triangle"
+                    size="medium"
+                />
+                <span
+                    >You won't receive notifications from the Health Gateway.
+                    You can update this from your Profile Page later.</span
+                >
+            </div>
+            <h4 class="text-h5 font-weight-bold mt-10 mb-2">
+                Terms of Service
+            </h4>
+            <HtmlTextAreaComponent
+                :input="termsOfService?.content"
+                class="mb-3"
+            />
```

Comment:
I suggest adding an `overflow-y-auto` class and setting a maximum height to make this take up less page space.

Reply:
![image](https://github.com/bcgov/healthgateway/assets/19548348/f355e72b-2df7-434d-bb61-6b68f7625216)